tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -587,6 +586,8 @@ def _execute_test_suite(self, suite: TestSuiteInfo) -> bool:
                             log.warning("%s failed and blocks the suite. Stopping suite execution.", test.name)
                             break
 
+                        ignore_error_rules.extend(load_test.get_ignore_error_rules())
```

Comment:
i think we need to add the rules even if we hit the break at 587

Reply:
you mean for all remaining for tests? or every test regardless? I don't see a point if test run failed.

Reply:
you are already adding the rule if the test failed.

the break is hit when the test failed **and** it blocks the suite

Reply:
you meant for that particular test? Don't care about reaming tests after hit break

Reply:
for the test, right? that is what you are trying to do. ignore rules are associates to specific tests

Reply:
when we block suite run, do we need to add rules for rest of the tests in suite?Lets' say suite has 1,2,3 test and 2 has failed and has flag to block suite. So ignore rule list should contain 1 and 2 but not 3 or all 1,2,3?

Reply:
it should not contain the rules for the tests that did not execute

Reply:
That's what I thought, thanks for the clarification